src/openrct2/core/FileStream.h

```diff
@@ -14,11 +14,11 @@
 
 namespace OpenRCT2
 {
-    enum
+    enum class FileMode
```

Comment:
Maybe best to add a type, just in case, lest we get weird compiler errors again.